lib/packmatic/field/central/file_header.ex

```diff
@@ -91,7 +91,9 @@ defimpl Packmatic.Field, for: Packmatic.Field.Central.FileHeader do
       encode_16(0),
       encode_16(0),
       encode_16(0),
-      encode_32(0),
```

Comment:
This should probably be inherited from some kind of package-level option instead of being a magic value

I do agree the original code with permission = 0 is obviously not good enough and should be fixed.###